frontend/playwright.config.ts

```diff
@@ -36,10 +36,8 @@ const config: PlaywrightTestConfig = {
     /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
     actionTimeout: 0,
     /* Base URL to use in actions like `await page.goto('/')`. */
-    // NOTE This gets envsubt'ituted right before executing the playwright tests
-    // so that we can inject the FRONTEND_API_URL from the environment
-    // at runtime.
-    baseURL: '${FRONTEND_API_URL}',
+    // default set with option to override via environment var
+    baseURL: process.env.FRONTEND_API_URL || 'http://localhost:8001',
```

Comment:
This should work since the tests are run locally and we won't need envsubst or sponge.

Reply:
Vite JS build system doesn't provide process.env so this won't work

Reply:
I agree it's superfluous at this point. Removed.